prov/rxm/src/rxm.h

```diff
@@ -488,12 +487,22 @@ struct rxm_rma_buf {
 	/* Must stay at top */
 	struct rxm_buf hdr;
 
-	struct fi_msg_rma msg;
-	struct rxm_iov rxm_iov;
-	struct rxm_rma_iov_storage rxm_rma_iov;
+	void *app_context;
+	uint64_t flags;
 
 	/* Must stay at bottom */
-	struct rxm_pkt pkt;
+ 	union {
+		struct rxm_pkt pkt;
+		struct {
+			struct fid_mr *mr[RXM_IOV_LIMIT];
+			uint8_t count;
+		} mr;
+		/* TODO: delete fields below */
+		struct {
+			struct rxm_iov rxm_iov;
+			struct rxm_rma_iov_storage rxm_rma_iov;
+		} def;
```

Comment:
Can the change that removes this be included in this commit?

Reply:
Will do that, but I need to double check that will not lead to a lot of merge conflicts